frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/CheckReAuthUserHandler.java

```diff
@@ -112,7 +112,10 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                     request.email(),
                     IpAddressHelper.extractIpAddress(input),
                     AuditService.UNKNOWN,
-                    PersistentIdHelper.extractPersistentIdFromHeaders(input.getHeaders()));
+                    PersistentIdHelper.extractPersistentIdFromHeaders(input.getHeaders()),
+                    AuditService.MetadataPair.pair(
+                            "number_of_attempts_user_allowed_to_login",
+                            configurationService.getMaxPasswordRetries()));
```

Comment:
I think here which thing it is is going to depend on whether it's a failed number of passwords or a failed number of email attempts - either can throw this error.